_summaries/zeiler2012adadelta_1.md

```diff
@@ -1,19 +1,48 @@
 ---
 layout: summary
 title: Summary
-paper: {{paper_tag}}
+paper: {{zeiler2012adadelta}}
 # Please fill out info below
-author: # Your GitHub id
-score: # How did you like this paper 0(dislike) to 10(love)
+author:  sritank
+score:  10
 ---
 
 TODO: Summarize the paper:
+
 * What is the core idea?
+- Adaptively change the learning rate in a way that it reduces oscillations i.e. slows down close to minimas, and is large otherwise.
+- Change step size based on the windowed accumulated gradient and also accumulated step sizes, which only require 1st order computations.
+- Also scale the step size by the accumulated previous step sizes, giving it the correct units as if it were a 2nd order adaptive method
```

Comment:
might be interesting to note that ADADELTA is a variation of ADAGRAD that address' the following 2 issues with ADAGRAD: (directly from the papaer) 
"1.the continual decay of learning rates throughout training, and 2) the need for a manually selected global learning rate."
EDIT: I see that you mentioned ADAGRAD below but might be good to bring up the 2 issues ADADELTA solves ADAGRAD does not (either verbally during presentation or in text)

Reply:
Added it to draft